File: 0191-number-of-1-bits/0191-number-of-1-bits.js
/**
 * @param {number} n
 * @return {number}
 */
var hammingWeight = function(n) {
    let a=""
    while(n>0){
        a+=n%2
        n=Math.floor(n/2)
    }
    console.log(a)
    let count=0
    for(let b of a ){
        if(b=="1"){
            count++
        }
    }
    return count 
};

File: 0191-number-of-1-bits/0191-number-of-1-bits.go

import "strconv"
func hammingWeight(n int) int {
    var res string 
    var count int 
    for n>0{
        res+=strconv.Itoa(n%2);
        n/=2;
    }
    // fmt.Println(res,len(res))
    for _,val := range res{
        // fmt.Println(i,string(val))
        if string(val)=="1"{
            count++
        }
    }
    return count 
}